src/main/cli/cli.c

```diff
@@ -1359,10 +1400,10 @@ static void cliSerialPassthrough(char *cmdline)
 #endif /* USE_PINIO */
         ) {
         // Register control line state callback
-        serialSetCtrlLineStateCb(cliPort, cbCtrlLine, (void *)(intptr_t)(pinioDtr));
+        serialSetCtrlLineStateCb(*(passThroughPorts[0]), cbCtrlLine, (void *)(intptr_t)(pinioDtr));
```

Comment:
How do you think this will work? We currently do not have a way to define DTR lines for non-VCP UARTS, and the callback needed to make this work for anything but VCP is also not defined.

Again, I think this functionality should just be declared incompatible with the two-UART forwarding that you are proposing, and treated as such.

Reply:
Sorry, the DTR doesn't test as well:(
I add a condition to avoid it while `leftSidePortId == -1`